### DISCUSSION TITLE
Deleted Project but domain is still attached to Vercel

### DISCUSSION THREAD

**Discussion by GinaCastromonte** [vercel/community#6479] [Help]
### Page to Investigate

www.runaboutfarm.org 

### Steps to Reproduce

I deleted a project about a week ago.  I deleted the domains attached and deleted the domain off my account.  However, the domain is still somehow attached to Vercel.  This is the error I got : 404: NOT_FOUND
Code: DEPLOYMENT_NOT_FOUND
ID: iad1::fb586-1712870934552-a6544fc219ce.   Thank you! @amyegan

**Comment by GinaCastromonte** [comment]
@amyegan  To continue from the previous thread, I did see that the dns still points to Vercel.  I've already changed the necessary DNS settings on Homestead .  Somehow it still is attached to Vercel even though I deleted the project and I deleted the domain off my profile.  Please help

**Reply by amyegan** [reply]
If you don't want the `www` version of your domain to point to Vercel, you need to remove the CNAME record. Right now the global DNS network has instructions to route `www.runaboutfarm.org` to Vercel, and that's why you're seeing a Vercel deployment error message.

I'm not familiar with Homestead configuration so I can't offer specific advice on that. But I'm sure their team can help you with any other DNS settings needed.

**Reply by GinaCastromonte** [reply]
Yea , Vercel's CNAME and everything has been deleted from the Homestead DNS configuration (the domain was purchased through them) .  So, this would be a Homestead issue even though Vercel's error is showing up attached to the domain?

**Reply by amyegan** [reply]
Yes, it's a DNS issue. The reason you're seeing a Vercel error message is because the DNS records are still pointing to Vercel but the domain isn't currently attached to any Vercel project. So when our system gets the request it doesn't have an associated deployment to show and instead gives you a 404 error.

Homestead should be able to help you get it sorted

**Reply by GinaCastromonte** [reply]
That makes sense, thank you so much for looking into this! Much appreciated
